### PR TITLE
Fix csv upload

### PR DESCRIPTION
**Pull request by acsauk**
After we started to persist order number on CSV upload the way we were processing entities caused a timeout. We didn't have a test with the number of rows that was representative of real-life so the problem was never caught.

I've ensured we're persisting and flushing in batches now and have disabled the Doctrine logger to keep memory usage low. I've also added a load test to our behat suite. It adds 10 seconds or so to the total run time but it's worth it for the confidence it gives us showing we can handle CSVs with 2000+ rows.

### CODE REVIEW

**Review by Gugandeep — APPROVED** [review]

File: serve-web/src/Service/OrderService.php [hunk 1]
```diff
@@ -119,11 +119,11 @@ public function upsert(
             $order = new $orderClass($client, $madeAt, $issuedAt, $orderNumber);
             $this->em->persist($order);
             $this->em->persist($client);
-            $this->em->flush();
-        } else {
```
Comment (Gugandeep) [inline serve-web/src/Service/OrderService.php:123]: Just checking you were meant to get rid of the else statement here, and not just the `flush()`s

Comment (acsauk) [reply serve-web/src/Service/OrderService.php:123]: I thought I'd pushed the wrong thing when I first looked at the diff too - there is an `if` there; its just the way git has shown the changes:

```php
        if (!$order) {
            $order = new $orderClass($client, $madeAt, $issuedAt, $orderNumber);
            $this->em->persist($order);
            $this->em->persist($client);
        }

        if ($order->getOrderNumber() !== $orderNumber){
            $order->setOrderNumber($orderNumber);
            $this->em->persist($order);
        }
```

Comment (acsauk) [reply serve-web/src/Service/OrderService.php:123]: And I misread your message 😂  time to log off! Yeah, it was intentional - I wanted to put some logic around it so we aren't updating every order number - just when it changes.

Comment (Gugandeep) [reply serve-web/src/Service/OrderService.php:123]: Ah must have missed it. Good to go then.

**Pull request merged**